### PR TITLE
Add macro log events to pool create, rename, and destroy operations

### PR DESCRIPTION
**Pull request by bgurney-rh**
Supersedes #1874

### CODE REVIEW

**Comment by bgurney-rh** [comment]
Rebased.

**Comment by jbaublitz** [comment]
This most recent change looks good! Once you're ready for another review, post the output you're proposing here too.

**Comment by bgurney-rh** [comment]
I almost forgot; here's the output for the last commit:
```
 INFO libstratis::dbus_api::api::manager_2_0::methods: preparing to create pool with name spool1
 INFO libstratis::dbus_api::api::manager_2_0::methods: pool with name spool1, pool UUID 16915724be8e43b6814d4cd9610723fd created
 INFO libstratis::dbus_api::pool::pool_2_0::methods: preparing to rename pool with name spool2, uuid 16915724be8e43b6814d4cd9610723fd
 INFO libstratis::dbus_api::pool::pool_2_0::methods: pool with name spool2, pool UUID 16915724be8e43b6814d4cd9610723fd renamed
 INFO libstratis::dbus_api::api::manager_2_0::methods: preparing to destroy pool with uuid 16915724be8e43b6814d4cd9610723fd
 INFO libstratis::dbus_api::api::manager_2_0::methods: pool with UUID 16915724be8e43b6814d4cd9610723fd destroyed
```

**Comment by bgurney-rh** [comment]
Rebased.

File: src/dbus_api/api/manager_2_0/methods.rs [hunk 1]
```diff
@@ -100,7 +101,11 @@ pub fn destroy_pool(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
         }
     };
 
-    let msg = match dbus_context.engine.borrow_mut().destroy_pool(pool_uuid) {
+    let mut engine = dbus_context.engine.borrow_mut();
+
+    let result = pool_op_logging!("preparing to destroy pool with uuid {}", pool_uuid.to_simple_ref(); engine.destroy_pool(pool_uuid));
```
Comment (jbaublitz) [inline src/dbus_api/api/manager_2_0/methods.rs]: Per @mulkieran's previous comment, it looks like "uuid" in the string should be capitalized.

File: src/dbus_api/pool/pool_2_0/methods.rs [hunk 2]
```diff
@@ -285,11 +285,11 @@ pub fn rename_pool(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
...
     let pool_uuid = get_data!(pool_path; default_return; return_message).uuid;
 
-    let msg = match dbus_context
-        .engine
-        .borrow_mut()
-        .rename_pool(pool_uuid, new_name)
-    {
+    let mut engine = dbus_context.engine.borrow_mut();
+
+    let result = pool_op_logging!("preparing to rename pool with name {}, uuid {}", new_name, pool_uuid.to_simple_ref(); engine.rename_pool(pool_uuid, new_name); "pool with name {}", new_name);
```
Comment (jbaublitz) [inline src/dbus_api/pool/pool_2_0/methods.rs]: Per @mulkieran's previous comment, it looks like "uuid" in the string should be capitalized.

File: src/dbus_api/api/manager_2_0/methods.rs (hunk 1 above)
Comment (bgurney-rh) [reply src/dbus_api/api/manager_2_0/methods.rs]: Oh, I missed one.  I'll get it.

**Comment by bgurney-rh** [comment]
Now all of the "UUID" occurrences in messages should be capitalized:
```
 INFO libstratis::dbus_api::api::manager_2_0::methods: preparing to create pool with name spool1
 INFO libstratis::dbus_api::api::manager_2_0::methods: pool with name spool1, pool UUID f81d7d6ec6c14c1abb3317d06def5a0c created
 INFO libstratis::dbus_api::pool::pool_2_0::methods: preparing to rename pool with name spool2, UUID f81d7d6ec6c14c1abb3317d06def5a0c
 INFO libstratis::dbus_api::pool::pool_2_0::methods: pool with name spool2, pool UUID f81d7d6ec6c14c1abb3317d06def5a0c renamed
 INFO libstratis::dbus_api::api::manager_2_0::methods: preparing to destroy pool with UUID f81d7d6ec6c14c1abb3317d06def5a0c
 INFO libstratis::dbus_api::api::manager_2_0::methods: pool with UUID f81d7d6ec6c14c1abb3317d06def5a0c destroyed
```

File: src/dbus_api/api/manager_2_0/methods.rs (hunk 1 above)
Comment (bgurney-rh) [reply src/dbus_api/api/manager_2_0/methods.rs]: Resolved.

File: src/dbus_api/pool/pool_2_0/methods.rs (hunk 2 above)
Comment (bgurney-rh) [reply src/dbus_api/pool/pool_2_0/methods.rs]: Resolved.

**Comment by mulkieran** [comment]
I just accidentally hit a button.

**Comment by bgurney-rh** [comment]
No problem; I'm going to close this and create a new PR.

**Pull request closed without merging**